src/components/common/ConnectWallet/MPCLogin.tsx

```diff
@@ -34,44 +37,48 @@ const MPCLogin = ({ onLogin }: { onLogin?: () => void }) => {
     <>
       {wallet && userInfo ? (
         <>
+          <Track {...CREATE_SAFE_EVENTS.CONTINUE_TO_CREATION} label={wallet.label}>
+            <Button
+              variant="outlined"
+              sx={{ px: 2, py: 1, borderWidth: '1px !important' }}
+              onClick={onLogin}
+              size="small"
+              disabled={loginPending}
+              fullWidth
+            >
+              <Box width="100%" display="flex" flexDirection="row" alignItems="center" gap={1}>
+                <img
+                  src={userInfo.profileImage}
+                  className={css.profileImg}
+                  alt="Profile Image"
+                  referrerPolicy="no-referrer"
+                />
+                <div className={css.profileData}>
+                  <Typography variant="subtitle2" fontWeight={700}>
+                    Continue as {userInfo.name}
+                  </Typography>
+                  <Typography variant="body2">{userInfo.email}</Typography>
+                </div>
+                <SvgIcon component={GoogleLogo} inheritViewBox fontSize="medium" sx={{ marginLeft: 'auto' }} />
+              </Box>
+            </Button>
+          </Track>
+        </>
+      ) : (
+        <Track {...MPC_WALLET_EVENTS.CONNECT_GOOGLE}>
```

Comment:
Wrapping the button with the `Track` component made it look smaller compared to the other button. We could make it a div or add 100% width on it.

<img width="378" alt="Screenshot 2023-10-13 at 09 25 49" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/232e7223-811d-43b8-8dee-5411f39a08c0">